EasyCommands/BlockHandlers/TerminalBlockHandlers.cs

```diff
@@ -66,6 +66,8 @@ public class TerminalBlockHandler<T> : BlockHandler<T> where T : class, IMyTermi
                 AddVectorHandler(Property.POSITION, block => block.GetPosition());
                 AddStringHandler(Property.NAME, b => b.CustomName, (b, v) => b.CustomName = v);
                 AddBooleanHandler(Property.SHOW, b => b.ShowInTerminal, (b, v) => b.ShowInTerminal = v);
+                AddStringHandler(Property.DATA, b => b.CustomData, (b, v) => b.CustomData = v);
+                AddStringHandler(Property.INFO, b => b.DetailedInfo);
```

Comment:
Definitely makes sense for TerminalBlockHandlers.

We should add similar ones to Cargo.  They can either use GetOwner().CustomData, or maybe there's a better property on inventories to use?

For displays, perhaps this returns the text?

```
set my display info to "Welcome!"
set my display data to "You are here"

Print "Display Text: " + my display info
```